airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -220,7 +221,8 @@ def run_query(
         configuration = {
             'query': {
                 'query': bql,
-                'useLegacySql': use_legacy_sql
+                'useLegacySql': use_legacy_sql,
+                'maximumBillingTier': "%d" % maximum_billing_tier
```

Comment:
Prefer .format() over `%`. Also, can does this work as an integer, rather than a string? Seems cleaner, if so.